en/flight_modes/mission.md

```diff
@@ -58,26 +58,56 @@ For more information about mission planning, see:
 * [Mission Planning](../flying/missions.md)
 * [Plan View](https://docs.qgroundcontrol.com/master/en/PlanView/PlanView.html) (*QGroundControl* User Guide)
 
+## Mission Feasibility Checks
+
+PX4 runs some basic sanity checks to determine if a mission is feasible when it is uploaded, and when the vehicle is first armed.
+If any of the checks fail, the user is notified and it is not possible to start the mission (on upload the mission may not be saved).
+
+A subset of the most important checks are listed below:
+
+- First mission item too far away from vehicle ([MIS_DIST_1WP](#MIS_DIST_1WP))

```

Comment:
FYI only, I moved the parameters into a separate section below alongside the other relevant parameters, they are just linked here.